test/e2e/config/e2e_conf.yaml

```diff
@@ -17,85 +17,96 @@ providers:
 - name: cluster-api
   type: CoreProvider
   versions:
-  - name: ${CAPI_FROM_RELEASE}
-    value: "https://github.com/kubernetes-sigs/cluster-api/releases/download/${CAPI_FROM_RELEASE}/core-components.yaml"
+  - name: v1.6.3
```

Comment:
We have already bumped main to CAPI 1.7.0-beta.0, should it be added here too?

Reply:
let's add a separate PR for that, to add an upgrade test for beta it needs more changes.

Reply:
Works for me.